docs/DevelopmentGuide.md

```diff
@@ -149,6 +121,88 @@ bundle exec rake dependency:lock['/app/gemfiles/ruby_3.3_stripe_*.gemfile']
 bundle exec rake dependency:lock['/app/gemfiles/ruby_3.3_stripe_latest.gemfile']
 ```
 
+**How to add new dependency group**
+
+1. Choose the Ruby runtime and group name for your tests. When defining a new group, follow the format `scope:group`.
+For example, if you want tests to run only on Ruby 3.3 for profiling, you can define this in the [`Matfixfile`](../Matrixfile).
```

Comment:
Also I think you started using "profiling" as an example... and then maybe changed your mind and used tracing below? :eyes:

Reply:
Fixed